Polyhedron/demo/Polyhedron/Scene_c3t3_item.cpp

```diff
@@ -1497,7 +1497,18 @@ void Scene_c3t3_item_priv::computeSpheres()
     if(red)
       c = QColor(Qt::red);
     else
-      c = spheres->color().darker(250);
+      c = QColor::fromHsv(120, 200,200,255);
```

Comment:
The `Scene_c3t3_item` already have a `color()`. I suggest to use it, so that we can change it using the usual way.